page-register.html

```diff
@@ -674,6 +674,10 @@ <h4 class="uppercase">Register Here</h4>
                                 <p class="mb0">By signing up, you agree to our
                                     <a href="#">Terms Of Use</a>
                                 </p>
+                                <hr style="margin: 8px auto;" />
```

Comment:
Do not add style directly to the tags, use css file for adding parameters.

Reply:
@simsausaurabh totally going with your words I would like to suggest not using inline definitions for the style. Please move them to `css/custom.css`.

Reply:
ok I'll